vuepress/docs/next/tutorials/devops/local-tips-and-tricks.md

```diff
@@ -0,0 +1,146 @@
+installing Entando the first time but then just steps #1 and #2 after subsequent Windows restarts. 
+
+1. Determine the interface address to Hyper-V, e.g. 32 below. Use cmd `route print` and look for the Interface entry for Hyper-V
+```
+Interface List
+ 32...00 15 5d 86 45 20 ......Hyper-V Virtual Ethernet Adapter
+```
+
+2. Using elevated privileges, add a persistent route to map an unclaimed IP (it's up to you to determine an IP unused on 
+your local network but ping is useful). We use -p to indicate a persistent route but since Hyper-V's interface can also change
+after restart it's not very persistent. 
+``` 
+route -p add [YOUR-PREFERRED-IP] mask 255.255.255.255 0.0.0.0 IF [HYPER-V-INTERFACE]
+route -p add 192.168.99.1 mask 255.255.255.255 0.0.0.0 IF 32
+```
+3. Verify your setting took by using `route print 192*`. This is useful after restart to see if the route needs
+to be added again.
+
+4. Next, configure your VM to claim the same address. Shell into the VM using `winpty multipass shell [YOUR-VM-NAME]`. 
+
+5. Change to the root user to make the following steps simpler: `sudo -i`
+
+6. Determine your network adaptor via `ip link`, e.g. eth0. You just need the name and it's often second in the list, after the loopback adapter.
+``` bash
+ubuntu@primary:~$ ip link
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
+    link/ether 00:15:5d:00:1a:0c brd ff:ff:ff:ff:ff:ff
+```
+
+7. `cd /etc/netplan`
+
+8. Create a netplan entry starting with 0 so it's loaded first: `vi 0-entando.yaml`  
+
+``` yaml
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    [YOUR-NETWORK-ADAPTOR]:
+      dhcp4: no
+      addresses: 
+        - [YOUR-PREFERRED-IP]/24
+```
+Example:
+
+``` yaml
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    eth0:
+      dhcp4: no
+      addresses: 
+        - 192.168.99.1/24
+```
+
+9. Apply the changes via `netplan apply`
+
+10. Verify connectivity via `ping 192.168.99.1` from the VM. You should get a response rather than a timeout.
+
+11. (Optional) Run a python server to verify you can access the VM from the host at `http://192.168.99.1:8000.` 
+It may take a minute or so before the server is ready. If you can access it you can use this IP 
+address to configure your Entando installation.
+
+12. You should now be able to install Entando using the preferred IP. If Entando was stalled 
+starting up (and already configured this way), it should continue starting up as soon as the 
+external address is functional again. 
+    
+### JHipster
+**Q:** How can I run JHipster on Windows? 
+
+**A:** 1) Run it under cmd or Powershell, 2) Using Git Bash, run `winpty jhipster.cmd`, 3) Use Ubuntu bash via WSL (1 or 2) or VM in Multipass
```

Comment:
Not sure if this displays as a bulleted list, but these should probably be bulleted items on multiple lines using markdown syntax.